frontend/app/src/components/history/transactions/TransactionEventNote.vue

```diff
@@ -129,7 +129,7 @@ const formatNotes = (
         class="d-inline-flex"
       >
         <hash-link
-          :class="$style['address']"
+          :class="[$style['address'], note.type === 'tx' && 'pl-2']"
```

Comment:
Since we have multiple properties an one that depends on a boolean can you change it to the object format?